lib/board_implementation/CFBoard.cpp

```diff
@@ -1,13 +1,94 @@
 #include <iostream>
 #include <stdint.h>
 #include <cctype>
+#include <vector>
 
 #include "CFBoard.h"
 
 
 void CFBoard::fromFEN(std::string FEN) {
     // configure the current board from input FEN
+    // Clearing the board
+    pawnBoard = 0LL;
+	kingBoard = 0LL;
+	queenBoard = 0LL;
+	rookBoard = 0LL;
+	knightBoard = 0LL;
+	bishopBoard = 0LL;
+	blackBoard = 0LL;
+	whiteBoard = 0LL;
+	turn = 0;
+    castleBools = 0;
```

Comment:
Inconsistent indentation.

Reply:
Changed all 4 spaces to tabs in the next commit.